webpack.common.js

```diff
@@ -0,0 +1,50 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2020, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const path = require('path');
+
+// eslint-disable no-undef
+const WEBPACK_COMMON_CONFIG = {
+    performance: {
+        hints: false
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                enforce: "pre",
+                use: ["source-map-loader"]
```

Comment:
@scottbell Do you know why would be using source-map-loader on this project but not in `nasa/openmct`?

Reply:
If I had to guess, it's because we want to be able to have source-maps from the pulled-in code from `openmct`.

Reply:
Per the [library](https://webpack.js.org/loaders/source-map-loader/):
> This loader is especially useful when using 3rd-party libraries having their own source maps.